english/khudi/buland_very_high.txt

```diff
@@ -0,0 +1,9 @@
+Make your self-confidence and capabilities so high,
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.